### ISSUE TITLE
Bus lane available in specific time duration

### ISSUE THREAD

**Issue by hsyfyc** [matsim-org/matsim-code-examples#891]
Hello community,
I would like to simulate bus lane in my scenario. I have found some ways such as duplicate the link in the network. 
However, I think these solutions are only suitable for the bus lane operated for the full day. In my scenario, the bus lane only operated for four hours (7:00-9:00 and 17:00-19:00), in other time the bus lane has no difference with normal lane. 
Is there any solution to solve this problem?

**Comment by JWJoubert** [comment]
Yes, @hsyfyc, I believe similar questions have been asked (and answered) before: see #383 and #884, as they are related. I know @rgraebe did this for his research project, but I do not recall if that is available in the public domain.

**Comment by hsyfyc** [comment]
Thank you for your reply. I had read #383 and I had the same problem as the last comment mentioned. 
Supposed that link 1 is from node 1 to node 2 with 3 lanes (capacity 3600) and one lane will be used for bus lane from 7am to 9am. In the `transitSchedule.xml` file, bus line 1 has been set to pass link 1. I can duplicate a link 2 from the same start node and end node, so that it can be used during 7am to 9am. Then the problem comes. 
First, for the bus has to pass the link 1 according to the settings, the allow mode for link 1 should be set as 'car,pt'. Then using `networkChangeEvent` I can only change the capacity of link 1 to 1200 and the capacity of link 2 to 2400 to simulate the situation. While this setting means that link 1 could still be used by car.
Second, if I change the `transitSchedule.xml` file, set the bus to pass link 2, it will result in error when link 2 is closed.
I think a good way to simulate this situation is change the allow mode in the `networkChangeEvent`, as #884 mentioned, but it is not available, so do you have some good solution?
